prime-router/src/testIntegration/resources/datatests/FHIR_to_HL7/mars_unwanted_condition_to_notes.hl7

```diff
@@ -0,0 +1,14 @@
+MSH|^~\&|CDC PRIME - Atlanta^2.16.840.1.114222.4.1.237821^ISO|Testing Lab^12D4567890^CLIA|||20240828174209+0000||ORU^R01^ORU_R01|bfe0d734-06a8-472e-bf6a-cad808150c4e|P|2.5.1|||NE|NE|USA|UNICODE UTF-8|ENG^English^ISO||PHLabReport-NoAck^ELR_Receiver^2.16.840.1.113883.9.11^ISO
+SFT|Centers for Disease Control and Prevention|0.2-SNAPSHOT|PRIME ReportStream|0.2-SNAPSHOT||20240827205908+0000
+PID|1||4a48b374-1aec-4b71-a595-41aa031ef42f^^^Testing Lab&12D4567890&CLIA^PI^Testing Lab&12D4567890&CLIA||Davar^Shallan^^^^^L||20060103|F||2106-3^white^^^^^2.5.1^^white|10 Pattern Way^^Newport^RI^02840^USA||(540) 867 5309^PRS^CP^^1^540^8675309^^^^^(540) 867 5309|||||||||N^Not Hispanic or Latino^HL70189^^^^2.9^^Not Hispanic or Latino||||||||N
+ORC|RE|bfe0d734-06a8-472e-bf6a-cad808150c4e^Testing Lab^12D4567890^CLIA|bfe0d734-06a8-472e-bf6a-cad808150c4e^Testing Lab^12D4567890^CLIA|||||||||1245319599^McTester^Phil^^^^^^NPI&2.16.840.1.113883.4.6&ISO^L^^^NPI||(530) 867 5309^WPN^PH^^1^530^8675309^^^^^(530) 867 5309|20240828174209+0000||||||Testing Lab^L^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^12D4567890|123 Beach Way^^Denver^CO^80210^USA|(530) 867 5309^WPN^PH^^1^530^8675309^^^^^(530) 867 5309|321 Ocean Drive^^Denver^CO^80210^USA
+OBR|1|bfe0d734-06a8-472e-bf6a-cad808150c4e^Testing Lab^12D4567890^CLIA|bfe0d734-06a8-472e-bf6a-cad808150c4e^Testing Lab^12D4567890^CLIA|96123-5^SARS-CoV-2 (COVID-19) RdRp gene [Presence] in Upper respiratory specimen by NAA with probe detection^LN|||20240828172709+0000|||||||||1245319599^McTester^Phil^^^^^^NPI&2.16.840.1.113883.4.6&ISO^L^^^NPI|(530) 867 5309^WPN^PH^^1^530^8675309^^^^^(530) 867 5309|||||20240828174209+0000|||F
+NTE|1||This is a note
+NTE|2|0|OBX filtered for identifier = 96123-5 - COVID-19; value = 260373001 - Detected. OBX was removed due to your jurisdictional reporting rules indicating this result is not reportable.
```

Comment:
This value should not be numeric, This should be one of the values in table HL70105 "O", "L" or "P"

Reply:
Can we also add a test where multiple observations are converted to notes?

Reply:
Ah, nice catch, thanks. Also added test with Flu A, Flu B, and COVID.